packages/grafana-ui/src/components/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Slider, Props } from './Slider';
+import { getTheme } from '../../themes';
+import { mount } from 'enzyme';
+
+const sliderProps: Props = {
+  orientation: 'vertical',
+  min: 10,
+  max: 20,
+  theme: getTheme(),
+};
+
+describe('Slider', () => {
+  it('renders correct html', () => {
+    const wrapper = mount(<Slider {...sliderProps} />);
+    expect(wrapper.html()).toMatchSnapshot();
```

Comment:
We have stopped using snapshots as they are mostly just annoying, better to test that the render did not fail. And check for something important you expect in the rendered output